### DISCUSSION TITLE
Possible to redirect static VRS IP to a Hostname such as ddns.net?

### DISCUSSION THREAD

**Discussion by BallerG** [vradarserver/vrs#107] [Q&A]
Hi

I have a static ip with Virgin Media, but I'd like to put my VRS online at the same time masking my external IP. I have tried to set up a Web Redirect hostname with noip.com but with no success

![image](https://user-images.githubusercontent.com/64506538/227567746-204e2160-7439-4da2-ac7d-e35e21248a72.png)

Has anyone had success with a web Redirect, and is this possible with VRS?

**Comment by rikgale** [comment]
Have you port forwarded your router? You'll need to set up port forwarding the port the VRS uses. Google how to do this as every router is different.

**Comment by BallerG** [comment]
The port is open in my router and I can access the external VRS ip from any decide outside my local network, such as over 4G

**Comment by rikgale** [comment]
Under the IP/target - you only want your static IP and not :8081/VirtualRadar

**Comment by BallerG** [comment]
I have tried that but it just returns the error

Hmm. We’re having trouble finding that site.

We can’t connect to the server at (removed).ddns.net.

**Comment by BallerG** [comment]
So after further playing around I have got it working on desktop

![image](https://user-images.githubusercontent.com/64506538/227723131-f114041a-3721-4d6b-a1ef-57af91eb33f4.png)

But mobile for some reason not zooming in correctly

![image](https://user-images.githubusercontent.com/64506538/227723230-47e58e22-a9ee-4940-a13d-0fb45dbf788d.png)

Connecting directly to the external VRS on mobile however does display correctly

![image](https://user-images.githubusercontent.com/64506538/227723335-32886fbf-7401-4a2d-a9a9-7b0f718bd170.png)

**Comment by vradarserver** [comment]
Regarding the mobile page being zoomed out - had you used the "request desktop version" browser feature on the page at some point in the past?

**Comment by BallerG** [comment]
I have never used request desktop on the phone
